src/pages/experiments/[experimentId]/plots-and-tables/dot-plot/index.jsx

```diff
@@ -0,0 +1,193 @@
+import React, { useEffect } from 'react';
+import {
+  Row,
+  Col,
+  Space,
+  Collapse,
+  Skeleton,
+} from 'antd';
+import { useSelector, useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import DotPlot from 'components/plots/DotPlot';
+import { loadPaginatedGeneProperties, loadGeneExpression } from 'redux/actions/genes';
+import { loadCellSets } from 'redux/actions/cellSets';
+import PlotStyling from 'components/plots/styling/PlotStyling';
+import SelectData from 'components/plots/styling/SelectData';
+import MarkerGeneSelection from 'components/plots/styling/MarkerGeneSelection';
+import Header from 'components/plots/Header';
+
+import {
+  updatePlotConfig,
+  loadPlotConfig,
+} from 'redux/actions/componentConfig';
+
+import Loader from 'components/Loader';
+
+const { Panel } = Collapse;
+const plotUuid = 'dotPlotMain';
+const plotType = 'dotPlot';
+const route = {
+  path: 'dot-plot',
+  breadcrumbName: 'Dot plot',
+};
+
+const plotStylingControlsConfig = [
+  {
+    panelTitle: 'Main schema',
+    controls: ['dimensions'],
+    children: [
+      {
+        panelTitle: 'Title',
+        controls: ['title'],
+      },
+      {
+        panelTitle: 'Font',
+        controls: ['font'],
+      },
+    ],
+  },
+  {
+    panelTitle: 'Axes and margins',
+    controls: ['axes'],
+  },
+  {
+    panelTitle: 'Colours',
+    controls: ['colourScheme', 'colourInversion'],
+  },
+  {
+    panelTitle: 'Legend',
+    controls: [
+      {
+        name: 'legend',
+        props: {
+          option: {
+            positions: 'top-bottom',
+          },
+        },
+      },
+    ],
+  },
+];
+
+const dotPlot = (props) => {
+  const { experimentId } = props;
+
+  const dispatch = useDispatch();
+  const config = useSelector((state) => state.componentConfig[plotUuid]?.config);
+  const fetching = useSelector((state) => state.genes.properties.views[plotUuid]?.fetching);
```

Comment:
you could do this instead:
`const {fetching: genesFetching, data: highestDispersionGenes} = useSelector(...`